mods/ra2/rules/soviet-infantry.yaml

```diff
@@ -92,12 +92,12 @@ shk:
 	Inherits: ^Infantry
 	Buildable:
 		Queue: Infantry
-		BuildAtProductionType: Infantry
```

Comment:
So why other doesn't have? I don't think so, this is from RA1 becuase there are kennels there which uses `BuildAtProductionType: Dog`.